lib/app/examples/http_request_example.nit

```diff
@@ -13,10 +13,15 @@
 # limitations under the License.
 
 # Example for the `app::http_request` main service `AsyncHttpRequest`
-module http_request_example
+module http_request_example is
+	app_name "app.nit HTTP ex."
```

Comment:
It was to long for the launcher on my phone anyway, so I dropped the `ex.` part.